src/attached_probe.cpp

```diff
@@ -328,6 +328,10 @@ resolve_offset(const std::string &path, const std::string &symbol, uint64_t loc)
   if (bcc_resolve_symname(path.c_str(), symbol.c_str(), loc, 0, nullptr, &bcc_sym))
      throw std::runtime_error("Could not resolve symbol: " + path + ":" + symbol);
 
+  // https://github.com/iovisor/bcc/blob/master/src/cc/bcc_syms.h#L98-L99
```

Comment:
```suggestion
  // Have to free sym.module, see:
  // https://github.com/iovisor/bcc/blob/ba73657cb8c4dab83dfb89eed4a8b3866255569a/src/cc/bcc_syms.h#L98-L99
```

Links to master tend to break when someone updates the file, you can select a permalink from the "menu" after selecting line numbers

Reply:
Ah, fair point. Will update. Thanks!